_inc/client/plans/product-selector.jsx

```diff
@@ -23,6 +25,14 @@ import { getSiteRawUrl, getUpgradeUrl, isMultisite } from '../state/initial-stat
 import { getProducts, isFetchingProducts } from '../state/products';
 
 class ProductSelector extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			selectedBackupType: 'real-time',
```

Comment:
Since this is a `Component`, I'd suggest moving this outside of `constructor`:

```
state = {
	selectedBackupType: 'real-time',
};
```

Then we won't even need to have a `constructor()` method at all.

Reply:
Sounds perfect. I removed the constructor in 8491b49.